code/game/gamemodes/cult/runes.dm

```diff
@@ -442,15 +442,18 @@ var/list/sacrificed = list()
 		return this_rune.fizzle(user)
 	var/mob/abstract/observer/ghost
```

Comment:
So, obviously this isn't caused by you. But this proc is super-weird.

1 - It casts itself as itself, I guess just so it can call itself `this_rune` for some reason. That's unnecessary, can just use src.
2 - The `ghost` var doesn't need to be given a type, it can just be `var/ghost = TRUE/FALSE`